src/main/kotlin/uk/gov/dwp/dataworks/logging/LogFields.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.dwp.dataworks.logging
+
+import java.net.InetAddress
+
+/**
+ * Object to hold the Application specific and Common fields that should be added to each of the log lines output by
+ * this library. Provides helper methods to return these fields as a set of json compliant Key:value pairs but without
+ * any wrapping (i.e no `{}`).
+ */
+object LogFields {
+    private val logFields = CommonLogFields.commonFields.toMutableMap()
+    val asJson = logFields.map { """ "${it.key}":"${it.value}" """.trim() }.joinToString(separator = ",")
+
+    fun get(logField: String): String {
+        return logFields.getValue(logField)
+    }
+
+    fun get(logField: LogField): String {
+        return get(logField.systemPropName)
+    }
+
+    fun put(logField: String, value: String) {
+        if(logField.isBlank() || value.isBlank()) {
+            throw IllegalArgumentException("Key and value must not be blank! Got: $logField:$value")

```

Comment:
If they can be blank can we put single quotes around them to show when they are?

Reply:
Yep, will update